mapping.json

```diff
@@ -1,38 +1,48 @@
+        { "url": "google-analytics.com/inpage_linkid.js", "surrogate": "inpage_linkid.js" },
+        { "url": "google-analytics.com/plugins/ga/inpage_linkid.js", "surrogate": "inpage_linkid.js" },
+        { "url": "google-analytics.com/cx/api.js", "surrogate": "api.js" },
+        { "url": "google-analytics.com/gtm/js", "surrogate": "gtm.js" }
     ],
     "googletagmanager.com": [
-        ["googletagmanager.com/gtm.js", "gtm.js"]
+        { "url": "googletagmanager.com/gtm.js", "surrogate": "gtm.js" }
     ],
     "googletagservices.com": [
-        ["googletagservices.com/gpt.js", "gpt.js"],
-        ["googletagservices.com/tag/js/gpt.js", "gpt.js"]
-    ],        
+        { "url": "googletagservices.com/gpt.js", "surrogate": "gpt.js" },
+        { "url": "googletagservices.com/tag/js/gpt.js", "surrogate": "gpt.js" }
+    ],
     "googlesyndication.com": [
-        ["googlesyndication.com/adsbygoogle.js", "adsbygoogle.js"],
-        ["googlesyndication.com/pagead/js/adsbygoogle.js", "adsbygoogle.js"]
+        { "url": "googlesyndication.com/adsbygoogle.js", "surrogate": "adsbygoogle.js" },
+        { "url": "googlesyndication.com/pagead/js/adsbygoogle.js", "surrogate": "adsbygoogle.js" }
     ],
     "doubleclick.net": [
-        ["doubleclick.net/instream/ad_status.js", "ad_status.js"],
-        ["doubleclick.net/tag/js/gpt.js", "gpt.js"]
+        { "url": "doubleclick.net/instream/ad_status.js", "surrogate": "ad_status.js" },
+        { "url": "doubleclick.net/tag/js/gpt.js", "surrogate": "gpt.js" }
     ],
     "scorecardresearch.com": [
-        ["scorecardresearch.com/beacon.js", "beacon.js"]
+        { "url": "scorecardresearch.com/beacon.js", "surrogate": "beacon.js" }
     ],
     "outbrain.com": [
-        ["outbrain.com/outbrain.js", "outbrain.js"]
+        { "url": "outbrain.com/outbrain.js", "surrogate": "outbrain.js" }
     ],
     "amazon-adsystem.com": [
-        ["amazon-adsystem.com/aax2/amzn_ads.js", "amzn_ads.js"]
+        { "url": "amazon-adsystem.com/aax2/amzn_ads.js", "surrogate": "amzn_ads.js" }
     ],
     "chartbeat.com": [
-        ["chartbeat.com/chartbeat.js", "chartbeat.js"],
-        ["chartbeat.com/js/chartbeat.js", "chartbeat.js"]
+        { "url": "chartbeat.com/chartbeat.js", "surrogate": "chartbeat.js" },
+        { "url": "chartbeat.com/js/chartbeat.js", "surrogate": "chartbeat.js" }
+    ],
+    "facebook.net": [
+        { "regexUrl": "facebook\\.net\\/[a-z_A-Z]+\\/sdk\\.js", "surrogate": "fb-sdk.js", "action": "block-ctl-fb" },
+        { "regexUrl": "facebook\\.net\\/[a-z_A-Z]+\\/all\\.js", "surrogate": "fb-sdk.js", "action": "block-ctl-fb" }
+    ],
+    "facebook.com": [
+        { "regexUrl": "facebook\\.com\\/[a-z_A-Z]+\\/sdk\\.js", "surrogate": "fb-sdk.js", "action": "block-ctl-fb" }
+    ],
+    "youtube.com": [
+        { "regexUrl": "(www\\.)?youtube(-nocookie)?\\.com\\/iframe_api", "surrogate": "youtube-iframe-api.js", "action": "block-ctl-yt" }
```

Comment:
`(www\\.)?` - do we need this part if it's optional?

Reply:
This is in the current YT surrogate URL (in the extension) so I just copied it over verbatim.

Reply:
cc @kzar - does this regex prefix do anything or can we drop it?

Reply:
Good point @kdzwinel, it looks like the check (`url.match(surrogate.rule))`) does not anchor to the start of the URL anyway, so I agree that we can drop the `(www\\.)?` prefix. [This is where the check takes place](https://github.com/duckduckgo/duckduckgo-privacy-extension/blob/develop/shared/js/background/tracker-utils.js#L84).